include/oneapi/dpl/experimental/kt/esimd_radix_sort_utils.h

```diff
@@ -148,9 +148,42 @@ __get_bucket(sycl::ext::intel::esimd::simd<_T, _N> __value, ::std::uint32_t __ra
     return sycl::ext::intel::esimd::simd<::std::uint16_t, _N>(__value >> __radix_offset) & __radix_mask;
 }
 
-template <typename T, bool __is_ascending>
-inline constexpr T __sort_identity =
-    __is_ascending? ::std::numeric_limits<T>::max() : ::std::numeric_limits<T>::lowest();
+template <typename T, bool __is_ascending, std::enable_if_t<::std::is_integral_v<T>, int> = 0>
+constexpr T
```

Comment:
inline ?

Reply:
It is excessive, because:
> A constexpr specifier used in a function or [static](https://en.cppreference.com/w/cpp/language/static) data member (since C++17) declaration implies inline

source: https://en.cppreference.com/w/cpp/language/constexpr